portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,43 +20,68 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import java.util.ArrayList;
+import java.util.*;
```

Comment:
I believe the Google Java style guide doesn't allow for wildcard imports

Reply:
got it, fixed!